extensions/vscode/.vscode/launch.json

```diff
@@ -6,12 +6,24 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "name": "Run Extension",
+      "name": "Debug with external API agent on 9001",
```

Comment:
Super nitpicky, but could we have the "Run Extension" option listed first so it is selected first as the debug option?

Reply:
Sure